pallets/file-system/src/tests.rs

```diff
@@ -293,18 +295,19 @@ fn bsp_volunteer_success() {
         // Go past genesis block so events get deposited
         System::set_block_number(1);
 
-        let user = RuntimeOrigin::signed(1);
-        let bsp = RuntimeOrigin::signed(2);
+        let owner = RuntimeOrigin::signed(AccountId32::new([1; 32]));
+        let bsp_account_id = AccountId32::new([2; 32]);
+        let bsp = RuntimeOrigin::signed(bsp_account_id.clone());
         let location = FileLocation::<Test>::try_from(b"test".to_vec()).unwrap();
-        let file_content = b"test".to_vec();
-        let fingerprint = BlakeTwo256::hash(&file_content);
+        // bypass the volunteer assignment threshold
```

Comment:
Add a `TODO` here, to remember to change this when we can actually verify the threshold properly